.circleci/config.yml

```diff
@@ -1269,6 +1269,46 @@ jobs:
           name: Verify Values Match
           command: |
             ./ops/scripts/ci-match-values-between-files.sh "<< parameters.file1_path >>" "<< parameters.pattern_file1 >>" "<< parameters.file2_path >>" "<< parameters.pattern_file2 >>"
+
+  test-proofs:
+    machine: true
+    steps:
+      - checkout
+      - check-changed:
+          patterns: contracts-bedrock/test/kontrol,contracts-bedrock/src/L1/OptimismPortal\.sol
+      - run:
+          name: Checkout submodules
+          command: |
+            git submodule sync
+            git submodule update --init --recursive
+      - run:
+          name: "Start Docker Container"
+          command: |
+            export KONTROL_VERSION=$(cat deps/kontrol_release)
```

Comment:
@tynes What's the preferred way to specify a kontrol version here? Looks like creating a `.kontrolrc` file at the repo root is the way to go, but wanted to confirm

Reply:
We currently use the dotfile approach like you mentioned. Its growing a ton with the number of tools but we should prefer to follow it